.env.example

```
@@ -37,6 +37,34 @@ AZURE_OPENAI_MODEL="gpt-4o-automatic"
+# The pgvector backend reuses KB_AZURE_OPENAI_* and the shared 768-d provider.
+# Worker controls below are Redis/DevCycle-overridable except scheduler cadence,
+# which is read once at process startup.
+MEMORY_EXTRACTION_INTERVAL_SECONDS=60
+MEMORY_EXTRACTION_BATCH_SIZE=20
+MEMORY_EXTRACTION_MAX_ATTEMPTS=3
+MEMORY_EXTRACTION_VISIBILITY_TIMEOUT_SECONDS=300
+MEMORY_EXTRACTION_RETRY_BASE_SECONDS=30
+MEMORY_MAX_FACTS_PER_USER=100
+# supermemory backend only:
+SUPERMEMORY_API_KEY=
+SUPERMEMORY_BASE_URL="https://api.supermemory.ai"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)